modules/api/test/LorisApiImagesTest.php

```diff
@@ -0,0 +1,724 @@
+                $response->getBody()->getContents()
+            ),
+            true
+        );
+
+        $this->assertSame(gettype($imagesArray), 'array');
+        $this->assertSame(gettype($imagesArray['Meta']), 'array');
+        $this->assertSame(gettype($imagesArray['Meta']['CandID']), 'string');
+        $this->assertSame(gettype($imagesArray['Meta']['Visit']), 'string');
+        $this->assertSame(gettype($imagesArray['Files']), 'array');
+        $this->assertSame(
+            gettype($imagesArray['Files']['0']['OutputType']),
+            'string'
+        );
+        $this->assertSame(
+            gettype($imagesArray['Files']['0']['Filename']),
+            'string'
+        );
+        $this->assertSame(
+            gettype($imagesArray['Files']['0']['AcquisitionType']),
+            'string'
+        );
+
+        $this->assertArrayHasKey('Meta', $imagesArray);
+        $this->assertArrayHasKey('CandID', $imagesArray['Meta']);
+        $this->assertArrayHasKey('Visit', $imagesArray['Meta']);
+        $this->assertArrayHasKey('Files', $imagesArray);
+        $this->assertArrayHasKey('OutputType', $imagesArray['Files']['0']);
+        $this->assertArrayHasKey('Filename', $imagesArray['Files']['0']);
+        $this->assertArrayHasKey('AcquisitionType', $imagesArray['Files']['0']);
+    }
+
+    /**
+     * Tests the HTTP GET request for the
+     * endpoint /candidates/{candid}/{visit}/images/filename
+     *
+     * @return void
+     */
+    public function testGetCandidatesCandidVisitImagesFilename(): void
+    {
+        $this->markTestSkipped('Missing data in docker image');
+    }
+
+    /**
+     * Tests the HTTP GET request for the
+     * endpoint /candidates/{candid}/{visit}/images/qc
+     *
+     * @return void
+     */
+    public function testGetCandidatesCandidVisitImagesFilenameQc(): void
+    {
+        $response = $this->client->request(
+            'GET',
+            "candidates/$this->candidTest/$this->visitTest/images/" .
+            "$this->imagefileTest/qc",
+            [
+                'headers'     => $this->headers,
+                'http_errors' => false,
+            ]
+        );
+        $this->assertEquals(200, $response->getStatusCode());
+        // Verify the endpoint has a body
+        $body = $response->getBody();
+        $this->assertNotEmpty($body);
+        $imagesArray = json_decode(
+            (string) utf8_encode(
+                $response->getBody()->getContents()
+            ),
+            true
+        );
+
+        $this->assertSame(
+            gettype($imagesArray),
+            'array'
+        );
+        $this->assertSame(
+            gettype($imagesArray['Meta']),
+            'array'
+        );
+        $this->assertSame(
+            gettype($imagesArray['Meta']['CandID']),
+            'string'
+        );
+        $this->assertSame(
+            gettype($imagesArray['Meta']['Visit']),
+            'string'
+        );
+        try {
```

Comment:
Why is this in a try catch?

Reply:
I can't remember why, it doesn't look to do anything. I'm removing it.